spec/javascript/utils/form/validations.test.js

```diff
@@ -0,0 +1,56 @@
+import validate from 'utils/form/validations'
+
+// Validations will return this message if failed, null if passed
+const FAILED_VALIDATION = 'failed validation'
+
+describe('validate', () => {
+  describe('isOldEnough', () => {
+    test('passes validation if DOB is more than 18 years ago', () => {
+      expect(validate.isOldEnough(FAILED_VALIDATION)(['2000', '01', '12'])).toEqual(null)
+    })
+    test('fails validation if DOB is less than 18 years ago', () => {
```

Comment:
one hopes this test will be updated before 2028, but to be proper about it, perhaps we could use moment to dynamically provide a date that is <18 years prior to the current date? not a priority obv

Reply:
fixed